### DISCUSSION TITLE
Change vehicle name?

### DISCUSSION THREAD

**Discussion by Kevr0n228** [Hyundai-Kia-Connect/kia_uvo#640] [Q&A]
Location: Canada
Vehicle: 2019 Kona EV

I just updated both HA and this integration to the latest version and it is now referring to my vehicle as "My Kona" where before it was just "Kona" - this has changed all of the sensor and control names to be "my_kona_..." instead of "kona_...", breaking all of the dashboard cards and automations that I had from before. Is there a way to set the name so its just "Kona" again?

New Sensor names:
<img width="376" alt="Screen Shot 2023-05-07 at 1 11 10 PM" src="https://user-images.githubusercontent.com/10468194/236692320-08343610-6a8f-4998-a15a-25bd7d2ff89b.png">
Old Sensor Names:
<img width="331" alt="Screen Shot 2023-05-07 at 1 10 36 PM" src="https://user-images.githubusercontent.com/10468194/236692314-abbe9cce-1c49-4b0c-bf37-c4234e056a5e.png">

**Comment by cdnninja** [comment]
I assume the car is called my kona in the native app.   Have you tried renaming the device in home assistant?

**Comment by Kevr0n228** [comment]
Seems it appends "My " to whatever i set the name as. Tried changing it
from and back to "Kona" but the entity names still have "my_" on them too.

**Reply by cdnninja** [reply]
Did it prompt you to update the entities?   If not somethings up with home assistant.   When it prompts tell it to rename them all.

**Comment by cdnninja** [comment]
As well just to confirm your saying it isn't called my kona in the native app either?

**Comment by Kevr0n228** [comment]
It did prompt me to rename all entities with the new device name, but
whatever I choose it prepends "my_". I checked the bluelink app, it's  just
called "Kona EV" there.
